cloudfront-cff-s3-staticsite-dotnet/README.md

```diff
@@ -0,0 +1,64 @@
+# AWS S3 Hosting Static Site with CloudFront
+
+This pattern helps you to deploy a CDK stack of CloudFront with Function and S3 bucket with Static Site. It uses MKDOCS to generate the static site. 
```

Comment:
Improve the readability of this paragraph, for example.

"This pattern helps you deploy a static site using Amazon CloudFront, AWS Lambda and AWS S3 with a CDK stack"